src/networks.json

```diff
@@ -307,6 +307,24 @@
     "rpc": ["https://rpc.testnet.moonbeam.network"],
     "explorer": "https://moonbase-blockscout.testnet.moonbeam.network/"
   },
+  "4689": {
```

Comment:
Can you make a separated PR to add these 2 networks?

Reply:
Modified this PR to "[iotex-balance] add support for iotx testnet"
I will create a new pull request for iotx mainnet

Reply:
@maswalker there are still changes for `xrc20-balance-of` in this pr.. can you please try moving them to new PR, Thanks